src/scan.js

```diff
@@ -1,7 +1,14 @@
 const path = require('path');
 const fs = require('fs');
 
-module.exports = function (pathName, dts) {
+//  ./button-next => ./ButtonNext
+const camelize = (str) => {
+  return str.replace(/(?:^\w|[A-Z]|\b\w)/g, function(letter, index) {
+    return index == 0 ? letter.toLowerCase() : letter.toUpperCase();
```

Comment:
Uppercase first character is probably a different "feature" than regular camel casing.